config/cloud.cfg.tmpl

```diff
@@ -121,7 +125,7 @@ cloud_final_modules:
 {% if variant in ["ubuntu", "unknown"] %}
  - ubuntu-drivers
 {% endif %}
-{% if variant not in ["freebsd"] %}
+{% if variant not in ["freebsd", "netbsd"] %}
```

Comment:
puppet works on FreeBSD, it's disabled, because 🤷‍♀️

Reply:
Agreed. I'm pretty sure it's the case for chef too. But this should be addressed in another patchset.